pkg/apiserver/apis/datapackaging/types.go

```diff
@@ -127,3 +132,10 @@ type ValuesSchema struct {
 	// +kubebuilder:pruning:PreserveUnknownFields
 	OpenAPIv3 runtime.RawExtension `json:"openAPIv3,omitempty"`
 }
+
+// IncludedSoftware contains the underlying Software Contents of a Package
+type IncludedSoftware struct {
+	Name        string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
```

Comment:
should we call this displayName since im assuming we are planning to use this for human oriented names vs like cert-manager.x.x

Reply:
I like `displayName` 👍